chapters/ch15.md

```diff
@@ -55,17 +55,17 @@ Image #3 above has both the Great Cat and the Blurry columns checked. Furthermor
 Although you may first formulate the categories (Dog, Great cat, Blurry) then categorize the examples by hand, in practice, once you start looking through examples, you will probably be inspired to propose new error categories. For example, say you go through a dozen images and realize a lot of mistakes occur with Instagram-filtered pictures. You can go back and add a new “Instagram” column to the spreadsheet. Manually looking at examples that the algorithm misclassified and asking how/whether you as a human could have labeled the picture correctly will often inspire you to come up with new categories of errors and solutions.
 ->
 
-Mặc dù bạn có thể tạo các hạng mục (Chó, Họ Mèo, Mờ) từ trước và sau đó phân loại các mẫu thủ công, thực tế trong quá trình phân tích mẫu, bạn có thể có những ý tưởng mới để tạo thêm các hạng mục. Ví dụ: bạn phân loại hàng chục bức ảnh và nhận ra nhiều lỗi xảy ra ở những tấm ảnh chỉnh bởi bộ lọc Instagram. Bạn có thể quay lại và thêm cột "Instagram" vào bảng biểu. Bằng cách nhìn vào từng mẫu mà thuật toán phân loại nhầm và đặt câu hỏi làm thế nào/liệu rằng con người có thể nhận diện mẫu này một cách chính xác, nhiều khả năng là bạn sẽ tìm được các lỗi và giải pháp mới.  
+Mặc dù bạn có thể tạo các hạng mục (Chó, Họ Mèo, Mờ) từ trước và sau đó phân loại các mẫu thủ công, thực tế trong quá trình phân tích mẫu, bạn có thể có những ý tưởng mới để tạo thêm các hạng mục. Ví dụ: bạn phân loại hàng chục bức ảnh và nhận ra nhiều lỗi xảy ra ở những tấm ảnh chỉnh bởi bộ lọc Instagram. Bạn có thể quay lại và thêm cột "Instagram" vào bảng biểu. Bằng cách nhìn vào từng mẫu mà thuật toán phân loại nhầm và đặt câu hỏi làm thế nào/liệu rằng con người có thể nhận dạng mẫu này một cách chính xác, nhiều khả năng là bạn sẽ tìm được các lỗi và giải pháp mới.  
```

Comment:
1. ở phía trên bạn đã dịch "great cat" thành "thú họ mèo" thì ở đây mình nghĩ nên giữ luôn như thế cho thống nhất và phân biệt luôn với mèo nuôi ở nhà.
2. "new categories of errors" = "hạng mục lỗi mới"??